vue-components/src/components/LookupInput.vue

```diff
@@ -140,10 +148,12 @@ export default defineComponent( {
 		onFocus(): void {
 			if ( this.canShowMenu( this.searchInput ) ) {
 				this.showMenu = true;
+				this.$emit( 'expanded', true );
```

Comment:
Is anything listening to this event?

Reply:
no, it's a leftover from something i tried. thanks for catching this. :sweat_smile: